modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -43,6 +43,10 @@
 }
 
 body .ddm-user-view-content {
+	.ddm-field-container {
```

Comment:
Try to keep classes as flat as possible.

Some useful information:
- https://thesassway.com/avoid-nested-selectors-for-more-modular-css/

Reply:
Thank you for the insight! I have updated the code to reflect these better css practices